src/storage-client/src/client.rs

```diff
@@ -169,41 +169,49 @@ impl RustType<ProtoRunIngestionCommand> for RunIngestionCommand {
     }
 }
 
-impl RustType<ProtoCreateSinkCommand> for CreateSinkCommand<mz_repr::Timestamp> {
-    fn into_proto(&self) -> ProtoCreateSinkCommand {
-        ProtoCreateSinkCommand {
+impl RustType<ProtoRunSinkCommand> for RunSinkCommand<mz_repr::Timestamp> {
+    fn into_proto(&self) -> ProtoRunSinkCommand {
+        ProtoRunSinkCommand {
             id: Some(self.id.into_proto()),
             description: Some(self.description.into_proto()),
+            update: self.update,
         }
     }
 
-    fn from_proto(proto: ProtoCreateSinkCommand) -> Result<Self, TryFromProtoError> {
-        Ok(CreateSinkCommand {
-            id: proto.id.into_rust_if_some("ProtoCreateSinkCommand::id")?,
+    fn from_proto(proto: ProtoRunSinkCommand) -> Result<Self, TryFromProtoError> {
+        Ok(RunSinkCommand {
+            id: proto.id.into_rust_if_some("ProtoRunSinkCommand::id")?,
             description: proto
                 .description
-                .into_rust_if_some("ProtoCreateSinkCommand::description")?,
+                .into_rust_if_some("ProtoRunSinkCommand::description")?,
+            update: proto.update,
         })
     }
 }
 
 /// A command that starts exporting the given sink description
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
-pub struct CreateSinkCommand<T> {
+pub struct RunSinkCommand<T> {
     pub id: GlobalId,
     pub description: StorageSinkDesc<MetadataFilled, T>,
+    pub update: bool,
 }
 
-impl Arbitrary for CreateSinkCommand<mz_repr::Timestamp> {
+impl Arbitrary for RunSinkCommand<mz_repr::Timestamp> {
     type Strategy = BoxedStrategy<Self>;
     type Parameters = ();
 
     fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
         (
             any::<GlobalId>(),
             any::<StorageSinkDesc<MetadataFilled, mz_repr::Timestamp>>(),
+            any::<bool>(),
         )
-            .prop_map(|(id, description)| Self { id, description })
+            .prop_map(|(id, description, update)| Self {
+                id,
+                description,
+                update,
+            })
```

Comment:
Tested by unit tests but not by end-to-end tests

Reply:
I believe that's common for these implementations.